### DISCUSSION TITLE
Welcome to dota-tutorial Discussions!

### DISCUSSION THREAD

**Discussion by SinZ163** [ModDota/dota-tutorial#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Post ideas in the Ideas category, and we will let the upvote / downvote mechanism try and get wisdom of the crowd to judge how good ideas are, but for now it may be better throwing stuff at discord and if it passes a sanity check there then post it here.

**Comment by LongtimeRYAN** [comment]
Hey is this a good place to put concepts that we think need to be addressed in the tutorial? Things like
-Basic movement
-Primary goals (destroy their towers/ancient/etc)
-spell usage
-item usage
-last hitting 
etc

**Reply by LongtimeRYAN** [reply]
if not here I could start a new discussion up to y'all

**Reply by SinZ163** [reply]
Things are still in flux and there is like 10 places where things can go, but a seperate discussion in the Ideas category would probably be better than this auto generated thread

**Reply by SinZ163** [reply]
in the 3 minutes since writing this, probably makes more sense to throw them at Slack's discord first and if they are vaguely good then put them here
